### PR TITLE
Ui: reset css

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- imports global reset CSS for legibility 
- ~adds concept of a `style-root` for components which want to be free of their parent's context~

## What is the value of this and can you measure success?

clarity and distinction

### CODE REVIEW

File: ui/src/components/head/reset.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+html {
+    box-sizing: border-box;
+}
+*,:after,:before {
+    box-sizing: inherit;
+}
+
+/* reset all inheritable properties */
+
+.reset {
```
Comment (gustavpursche) [inline ui/src/components/head/reset.css]: Do we really need this class? I mean, it doesn't reset everything and therefore it's a bit misleading ...

**Comment by SiAdcock** [comment]
It will be interesting to see how well this concept works with themes or tones that are passed down from a parent. Not really something to think about right now, but definitely before we roll out too many other components that depend on this behaviour

**Comment by sndrs** [comment]
@gustavpursche @SiAdcock removed `style-root` as per our IRL FF2FF

**Review by gu-stav — APPROVED** [review]
Let's go with this solution and have a look into the actual reset.css code, before we start building bigger things. 🚢

**Pull request merged**